OracleSOASuite/dockerfiles/12.2.1.4/README.md

```diff
@@ -1,82 +1,186 @@
 SOA on Docker
 =============
 
-Sample Docker configurations to facilitate installation, configuration, and environment setup for Docker users. This project includes quick start dockerfiles for Oracle SOA 12.2.1.4.0 based on Oracle Linux 7, Oracle JRE 8 (Server) and Oracle Fusion Middleware Infrastructure 12.2.1.4.0.
+Sample Docker configurations to facilitate installation, configuration, and environment setup for Docker users. This project includes quick start dockerfiles for Oracle SOA 12.2.1.4.0 based on Oracle Linux 7, Oracle JRE 8 (Server), and Oracle Fusion Middleware Infrastructure 12.2.1.4.0.
```

Comment:
Please remove "Docker" from the beginning of this line, and change "dockerfiles" to `Dockerfiles` (wrapped in backtick quotes). Also, it should be "Oracle Server JRE 8"